api/v1alpha1/multiclusterservice_types.go

```diff
@@ -15,7 +15,7 @@
 package v1alpha1
 
 import (
-	sveltosv1beta1 "github.com/projectsveltos/addon-controller/api/v1beta1"
+	addontontrollerv1beta1 "github.com/projectsveltos/addon-controller/api/v1beta1"
```

Comment:
```suggestion
	addoncontrollerv1beta1 "github.com/projectsveltos/addon-controller/api/v1beta1"
```
Applies to the rest of the imports.